Remove the specification that the blockID has to be validated (#42)
LiskHQ/lips: proposals/lip-0020.md

### BlockIDs in JSON Objects

> In a JSON object representing a block, all properties that contain a blockID have to be represented as a hexadecimal string. The properties of a block that contain a blockID are the `previousBlock` and the `id` property, whereby only the `previousBlock` property is required.

Reviewer: I would not even mention `id` as a property of a block JSON object. Maybe something like: 
```suggestion
In a JSON object representing a block, the property `previousBlock`, containing a blockID, has to be represented as a hexadecimal string.
```

Also, going through the LIP I noticed a possible typo:
```suggestion
In the proposed protocol, the blockID of a transaction B is computed as follows:
```

Finally, I would change the very last sentence in the backwards compatibility section:
```suggestion
...since the format of the `previousBlock` property change.
```

Author: all changes applied
